.github/workflows/cloud_run.yml

```diff
@@ -109,6 +115,21 @@ jobs:
             defaultFlags=$(echo "$defaultFlags --tag rc-candidate --no-traffic")
           fi
 
+          if [[ ${{ inputs.VPC_CONNECTOR }} = true ]]; then
+            gcloud services enable vpcaccess.googleapis.com
+            
+            emptyResponse=$(echo "[]")
+            connectorListResult=$(gcloud compute networks vpc-access connectors list --region=${{ inputs.REGION }} --filter="projects/${{ secrets.PROJECT_ID }}/locations/${{ inputs.REGION }}/connectors/${{ secrets.PROJECT_ID }}-vpc-connector" --format=json)
+            if [ $connectorListResult == $emptyResponse ]; then
+              gcloud compute networks vpc-access connectors create ${{ secrets.PROJECT_ID }}-vpc-connector \
+                --network default \
+                --region ${{ inputs.REGION }} \
+                --range 10.8.0.0/28
+            fi
+
+            defaultFlags=$(echo "$defaultFlags" --vpc-connector=${{ secrets.PROJECT_ID }}-vpc-connector)
```

Comment:
If you put the server in the VPC will be able to be accessed by frontend?

Reply:
yes